### DISCUSSION TITLE
`dependencyDashboardAutoclose` not working when `lockFileMaintenance` is enabled

### DISCUSSION THREAD

**Discussion by CarrotManMatt** [renovatebot/renovate#33242] [Request Help] [answered]
### How are you running Renovate?

A Mend.io-hosted app

### Please tell us more about your question or problem

My `renovate.json` config contains `"dependencyDashboardAutoclose": true` along with `"lockFileMaintenance": {"enabled": true}}`. I believe this is causing the dependency dashboard to never be closed due to the checkbox that allows running the scheduled lock-file maintenance task to be run. The link to my dependency dashboard is here: https://github.com/CarrotManMatt/static-websites/issues/19

**Comment by rarkins** [comment, marked as answer]
This is expected behavior, not easy or desirable to change

**Reply by ssbarnea** [reply]
Considering that renovate is already able to reopen closed issues, I wonder why this is not easy to fix. Regarding "desirable" part, I can only the marketing aspect as being a reason for keeping the issue open.

**Comment by CarrotManMatt** [comment]
Many thanks for the swift reply.💚

Is there any way to create a configuration with the following requirements then?

1. Lock file maintenance is scheduled to run at the default interval/schedule
2. Lock file maintenance can be manually requested from the Mend.io application web dashboard
3. The checkbox to request manual lock file maintenance does **not** appear on the dependency dashboard GitHub issue (thus allowing it to be auto closed when all dependency PRs have been merged)

**Reply by rarkins** [reply]
No, there's not

**Reply by CarrotManMatt** [reply]
Cheers, thanks for letting me know. And thank your for helping to develop such a great tool!
